### PR TITLE
MHV-42360: Pdf scaffold helper created

### PR DESCRIPTION
**Pull request by mattwrightva**
## Summary
I created a pdf generator scaffold generator function to reduce repeated code in pdf generator implementations. 

## Related issue(s)
https://jira.devops.va.gov/browse/MHV-42360

## Testing done
none yet

## Screenshots
No UI

## What areas of the site does it impact?
Platform pdf generator

## Acceptance criteria
Pdf generator scaffold generator function reduces repeated code. 

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### CODE REVIEW

File: src/applications/mhv/shared/util/helpers.js
Comment (acrollet) [inline src/applications/mhv/shared/util/helpers.js:30]: Can you add the bold formatting in the VCL banner? Unless it's been removed from the comps and I missed it.

File: src/applications/mhv/shared/util/helpers.js
Comment (acrollet) [inline src/applications/mhv/shared/util/helpers.js:34]: Should this use the platform dateFormatLong util? Fine either way but if it matches the format might as well.

**Review by acrollet — APPROVED** [review]
Approved pending tests and resolving feedback since I'll be out tomorrow, don't want to hold this up. 🙂

File: src/applications/mhv/shared/util/helpers.js
Comment (mattwrightva) [reply src/applications/mhv/shared/util/helpers.js:30]: Does PDFkit accept inline formatting?

Comment (mattwrightva) [reply src/applications/mhv/shared/util/helpers.js:30]: Oh yes, forgot. Thanks!

Comment (mattwrightva) [reply src/applications/mhv/shared/util/helpers.js:30]: A very clever way to implement bold text in the header banner!

File: src/applications/mhv/shared/util/helpers.js
Comment (mattwrightva) [reply src/applications/mhv/shared/util/helpers.js:34]: Definitely, another oversight. Thanks!

Comment (mattwrightva) [reply src/applications/mhv/shared/util/helpers.js:34]: And thank you for looking at this so late!

**Pull request merged**
